src/py/flwr/simulation/ray_transport/ray_client_proxy.py

```diff
@@ -127,15 +53,19 @@ def __init__(
         self, client_fn: ClientFn, cid: str, actor_pool: VirtualClientEngineActorPool
     ):
         super().__init__(cid)
-        self.client_fn = client_fn
+
+        def _load_app() -> ClientApp:
+            return ClientApp(client_fn=client_fn)
```

Comment:
Wouldn't this part need to support loading a `ClientApp` from a `module:attribute` `str` like `client:app`?

Reply:
no, here we are mirroring this: https://github.com/adap/flower/blob/314492cc64eff9689d98c2d22eba348d801c4d28/src/py/flwr/client/app.py#L379
for the upcoming VCE+DriverAPI we are [indeed doing](https://github.com/adap/flower/blob/3b64ff9d3fe7b6080db0b3b1e31bd23b25c6317e/src/py/flwr/server/superlink/fleet/vce/vce_api.py#L202C26-L202C41) what you suggest. The new VCE won't make use of these proxies.

Reply:
Thanks for the clarification